jena-core/src/main/java/org/apache/jena/rdf/model/impl/ModelCom.java

```diff
@@ -1604,9 +1542,8 @@ public GraphListener adapt( final ModelChangedListener L )
       { return new ModelListenerAdapter( this, L ); }
 
       @Override
-      public Model notifyEvent( Object e )
-      {
-          getGraph().getEventManager().notifyEvent( getGraph(), e );
+      public Model notifyEvent(Object e) {
+          getGraph().getEventManager().notifyEvent(getGraph(), e);
           return this;
       }
-}
+  }
```

Comment:
GH UI shows this as the last line of the file, I think. If it really is, then maybe this space prefix is not needed?

Reply:
Good ctach. There's a general misalignment in the latter part of the file. I'll fix that and use the opportunity to reformat he the code.